appengine/pond/duck/online/js/online.js

```diff
@@ -90,6 +88,12 @@ Pond.Duck.Online.init = function() {
     // Show Blockly editor tab.
     Pond.Duck.changeTab(Pond.Duck.TAB_INDEX.BLOCKLY);
   }
+  BlocklyInterface.workspace.addChangeListener(function(){
+    var saveBtn = document.getElementById('saveButton');
+    if (saveBtn) {
+      saveBtn.disabled = false;
+    }
```

Comment:
Blockly Games has some functionality implemented for whether code has changed in:
`BlocklyInterface.codeChanged`
We'd probably want to test it to make sure it works for us. I'm fine with changing it in a future PR and leaving this for now.

Reply:
Ok sweet I didn't realize that existed, will add in the PR I am currently working on : )